runelite-client/src/main/java/net/runelite/client/ui/overlay/components/BackgroundComponent.java

```diff
@@ -39,30 +39,42 @@
 public class BackgroundComponent implements RenderableEntity
 {
 	private static final int BORDER_OFFSET = 2;
-	private static final Color BACKGROUND_COLOR = new Color(70, 61, 50, 156);
-	private static final Color OUTSIDE_STROKE_COLOR = new Color(56, 48, 35, 255);
-	private static final Color INSIDE_STROKE_COLOR = new Color(90, 82, 69, 255);
+
+	@Setter
+	private Color backgroundColor = new Color(70, 61, 50, 156);
```

Comment:
I feel like this should be stored as constants somewhere, as it is now used at 3 different places with same value.